org.eclipse.winery.model.csar.toscametafile/src/main/java/org/eclipse/winery/model/csar/toscametafile/TOSCAMetaFileAttributes.java

```diff
@@ -23,6 +23,7 @@
     final public static String TOSCA_META_VERSION_VALUE = "1.0";
     final public static String CSAR_VERSION = "CSAR-Version";
     final public static String CSAR_VERSION_VALUE = "1.0";
+    final public static String YAML_CSAR_VERSION_VALUE = "1.1";
```

Comment:
Should we use `1.3`?! ... or is this another version?

Reply:
This describes the Metafile version, taken from the spec:
https://docs.oasis-open.org/tosca/TOSCA-Simple-Profile-YAML/v1.3/os/TOSCA-Simple-Profile-YAML-v1.3-os.html#_Toc454457870